platform/security/common/src/main/java/org/codice/ddf/security/common/jaxrs/RestSecurity.java

```diff
@@ -142,7 +142,7 @@ public static String deflateAndBase64Encode(String value) throws IOException {
   }
 
   public static String inflateBase64(String base64EncodedValue) throws IOException {
-    byte[] deflatedValue = base64Decode(base64EncodedValue);
+    byte[] deflatedValue = Base64.getMimeDecoder().decode(base64EncodedValue);
```

Comment:
Are we certain that this should be decoding using `ISO-8859-1` and not `UTF-8`? The decode method below is using `UTF-8`